pkg/cri/server/metrics.go

```diff
@@ -77,7 +77,7 @@ func init() {
 			Buckets: prom.DefBuckets,
 		},
 	)
-
+	ns.Add(imagePullThroughput)
```

Comment:
https://github.com/containerd/containerd/blob/988ee8ffef1e756039fe05539be3398e70ed3e0c/pkg/cri/server/metrics.go#L73-L80

Collectors added separately to metrics.Namespace need to manually add `metrics.Namespace` information.
```go
func init() {
        var (
                namespace = "containerd"
                subsystem = "cri"
                labels    = metrics.Labels(nil)
        )
        ns := metrics.NewNamespace(namespace, subsystem, labels)
        ...
        imagePullThroughput = prom.NewHistogram(
                prom.HistogramOpts{
                        Namespace: namespace,
                        Subsystem: subsystem,
                        Name:      "image_pulling_throughput",
                        Help:      "image pull throughput",
                        Buckets:   prom.DefBuckets,
                },
        )

        ns.Add(imagePullThroughput)
}